backend/manager/modules/vdsbroker/src/main/java/org/ovirt/engine/core/vdsbroker/monitoring/VmAnalyzer.java

```diff
@@ -287,6 +287,7 @@ void proceedDownVm() {
                 return;
             }
         }
+        vdsManager.unpinVmCpus(getVmId());
```

Comment:
this code indeed simplifies things but in order to be consistent with the comment above, we better destroy the vm first and then unpin it